server_benchmark.py

```diff
@@ -0,0 +1,101 @@
+"""
+Stand-alone benchmark for the GA4GH reference implementation.
+
+Assumes that wormtable sample data is installed at ./ga4gh-example-data.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+import time
+import argparse
+
+import ga4gh.backend
+import ga4gh.protocol as protocol
+import ga4gh.datamodel.variants as variants
+
+
+def _heavyQuery():
+    """
+    Very heavy query: all calls on chromosome 2
+    (11 pages, 90 seconds to fetch the entire thing
+    on a high-end desktop machine)
+    """
+    request = protocol.GASearchVariantsRequest()
+    request.referenceName = '2'
+    request.variantSetIds = ['1000g_2013']
+    request.callSetIds = None
+    request.pageSize = 100
+    request.end = 100000
+    return request
+
+
+def timeOneSearch(queryString):
+    """
+    Returns (search result as JSON string, time elapsed during search)
+    """
+    startTime = time.clock()
+    resultString = backend.searchVariants(queryString)
+    endTime = time.clock()
+    elapsedTime = endTime - startTime
+    return (resultString, elapsedTime)
```

Comment:
The brackets aren't actually needed here. Feel free to keep them if you think it improves readability --- just thought I'd point it out.